crates/search/src/project_search.rs

```diff
@@ -168,6 +169,8 @@ pub struct ProjectSearchView {
     replace_enabled: bool,
     included_opened_only: bool,
     _subscriptions: Vec<Subscription>,
+    search_debounce: Option<Task<()>>,
+    prevent_focus_results: bool,
```

Comment:
Is there a better way to handle this state?

Reply:
Been trying to solve the issue of keeping the prevent_focus_results as struct field. I don't really like having it there but can't quite figure out how to make it proper. Any suggestions would be welcome.